lib/wda/webdriveragent.js

```diff
@@ -20,6 +20,9 @@ const WDA_BASE_URL = 'http://localhost';
 
 const SHARED_RESOURCES_GUARD = new AsyncLock();
 
+const SAFARI_BUNDLE_ID = 'com.apple.mobilesafari';
+const WEBDRIVERAGENT_TEST_BUNDLE_ID = 'com.facebook.wda.runner';
```

Comment:
I assume the bundle id is changeable and may differ from that

Reply:
I can return the installed one but it shouldn’t change